### PR TITLE
Feat loading screen

### PR DESCRIPTION
**Pull request by psk001**

### CODE REVIEW

File: src/components/loadingScreenUp.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+const LoadingScreenUp = () => {
+  return (
+    <div className="h-[830px] w-[390px] rounded-tl-[33px] rounded-tr-[33px]">
+      <div className="h-[436px] w-[390px] rounded-bl-[30px] rounded-br-[170px] rounded-tl-[33px] rounded-tr-[33px] bg-[#FA5757]">
+        <img
+          className="absolute left-[155px] top-[225px] h-[225px] opacity-40"
+          src={'/assets/tlogo.svg'}
+        ></img>
+        <div className="relative left-[15px] top-[189px] h-[120px] w-[258px] font-['SF_Pro_Text'] text-[#560F0F] text-[50.67px] font-bold leading-[60.46px] tracking-[-0.54px]">
+          Prepping your repps!
```
Comment (pranav-9) [inline src/components/loadingScreenUp.js:12]: type repps should be "reps" @psk001

File: src/pages/index.js
Comment (VrindaTyagi) [inline src/pages/index.js:1]: @psk001 hey, please make sure the file names and component names are exactly the same and in camel casing starting from capitalizing the first letter

File: src/pages/loadingPage.js
Comment (VrindaTyagi) [inline src/pages/loadingPage.js:1]: @psk001 i believe we can simplify this conditional rendering, 

 {!showLower ? <LoadingScreenUp /> : null}
      {showLower && <LoadingScreenDown />}

File: src/components/loadingScreenUp.js (hunk 1 above)
Comment (VrindaTyagi) [reply src/components/loadingScreenUp.js:12]: @psk001 so in tailwind you can actually refer the docs and see what unit is 15px and then replace that with a single number. That will greatly improve how to code looks.

**Pull request merged**